README.md

```diff
@@ -135,25 +163,34 @@ assertThatBackButtonClosesTheApp();
 assertDrawable(R.id.image_view, R.drawable.ic_barista);
 ```
 
-## Dealing with the runtime permissions dialog
+## Runtime Permissions
 
 The new Marshmallow permissions system requires checking for permissions at runtime. As Espresso can't interact with the system dialog, Barista offers a way to allow permissions when needed.
 
 ```java
 PermissionGranter.allowPermissionsIfNeeded(Manifest.permission.GET_ACCOUNTS);
 ```
 
-## Resetting the app's data before running each test
+## Useful test rules
+Barista includes a set of useful rules test rules to help your tests.
+
+### Reseting app data
 
 As tests should be isolated, they need to set the environment before running. Espresso doesn't help achieving it but Barista offers a set of rules to clear the app's data before running each test.
 
 ```java
-@Rule public ClearPreferencesRule clearPreferencesRule = new ClearPreferencesRule(); // Clear all app's SharedPreferences
-@Rule public ClearDatabaseRule clearDatabaseRule = new ClearDatabaseRule(); // Delete all tables from all the app's SQLite Databases
-@Rule public ClearFilesRule clearFilesRule = new ClearFilesRule(); // Delete all files in getFilesDir() and getCacheDir()
+// Clear all app's SharedPreferences
+@Rule public ClearPreferencesRule clearPreferencesRule = new ClearPreferencesRule();
+
+// Delete all tables from all the app's SQLite Databases
+@Rule public ClearDatabaseRule clearDatabaseRule = new ClearDatabaseRule();
+
+// Delete all files in getFilesDir() and getCacheDir()
+@Rule public ClearFilesRule clearFilesRule = new ClearFilesRule();
 ```
 
-## Dealing with Flaky tests
+
+### Dealing with Flaky tests
```

Comment:
The FlakyActivityTestRule is deprecated in favor of FlakyTestRule+ActivityTestRule. But when writing the readme, it seems easier to just use FlakyActivityTestRule.

Should we un-deprecate it? Or should I write in the readme the best way to use the new one with a RuleChain? I'm having second thoughts.

Reply:
I think that the best thing we can do is to just aim everyone to use the BaristaRule. Actually, it's what I do in all projects I work and what I recommend to everyone. We could just change the way we explain it at the readme... Right now, the most explained rule is the Flaky one, but just because it was the first one that we written. Now, we can introduce the BaristaRule first, and then explain the Flaky rule and the other ones in another file. Well! But first, let's merge this PR :·D

Reply:
I don't think BaristaRule is customizable enough yet :(
For example we need to control excludes from the database rule, or add more custom rules (IdlingResources, memory cache clearing...) and they need to be inserted in a specific part of the RuleChain.

So... yes, I agree that the right approach is to make BaristaRule satisfy everyone's needs, but it's not ready. We need to improve it before.

Reply:
Then, you're answering your own question 👅 You should explain how to use it with a chain.